assets/assets/exceptions.py

```diff
@@ -28,3 +28,11 @@ class FileConversionError(Exception):
     """
 
     pass
+
+
+class AssetsUnsupportedFileFormat(Exception):
+    pass
+
+
+class AssetsMaxMoleculesCountExceeded(Exception):
```

Comment:
Should we specify `Assets` at the beginning of the exception name? It is located in assets microservice anyway